tools/runtime-osgi/pom.xml

```diff
@@ -60,13 +60,6 @@
 						<Embed-Transitive>true</Embed-Transitive>
 					</instructions>
 				</configuration>
-				<dependencies>
-					<dependency>
-						<groupId>biz.aQute.bnd</groupId>
-						<artifactId>bndlib</artifactId>
-						<version>2.4.0</version>
-					</dependency>
-				</dependencies>
```

Comment:
I'm not sure what this dependency was there for in the first place, do you know what it is and why it can be removed?

Reply:
Honestly, I removed that dependency without much thought and, in particular, I can't answer your questions. My idea was to get the updated `bndlib` used by the updated version of `maven-bundle-plugin`.